packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -212,11 +212,13 @@ export class DataVisualizerGraph
 		const result: Record<string, RootHandleNode> = {};
 		await Promise.all(
 			rootDataEntries.map(async ([key, value]) => {
-				const fluidObjectId = await this.registerVisualizerForHandle(value.handle);
-				result[key] =
-					fluidObjectId === undefined
-						? unknownObjectNode
-						: createHandleNode(fluidObjectId);
+				if (value.handle !== undefined) {
```

Comment:
In the case where `value.handle` is undefined, we should do 2 things (in my opinion):
1. Log an error to the console
2. Set `result[key]` to `unknownObjectNode`

This way we give ample warning to the user that they've configured something wrong.